pulpcore/cli/common/generic.py

```diff
@@ -390,10 +396,13 @@ def label_command(**kwargs: Any) -> click.Command:
     if "name" not in kwargs:
         kwargs["name"] = "label"
     decorators = kwargs.pop("decorators", [name_option, href_option])
+    need_plugins = kwargs.pop("need_plugins", [PluginRequiredVersion("core", "3.10.0")])
 
     @click.group(**kwargs)
-    def label_group() -> None:
-        pass
+    @pass_pulp_context
+    def label_group(pulp_ctx: PulpContext) -> None:
+        for item in need_plugins:
+            pulp_ctx.needs_plugin(item.name, item.min, item.max)
```

Comment:
Would this work?
```suggestion
        for item in need_plugins:
            pulp_ctx.needs_plugin(*item)
```

or this?
```suggestion
        [pulp_ctx.needs_plugin(*item) for item in need_plugins]
```

Reply:
1. Yes. 2. Yes, but mypy does not like it since `needs_plugin` doesn't return anything.

Reply:
yep, i agree the latter is rather unintuive. Usually you use list comprehensions that do _not_ have side effects. This would be only using the side effect.